src/azure-cli/azure/cli/command_modules/serviceconnector/_resource_config.py

```diff
@@ -1104,3 +1104,10 @@ class CLIENT_TYPE(Enum):
 SUPPORTED_CLIENT_TYPE[RESOURCE.ContainerApp] = SUPPORTED_CLIENT_TYPE[RESOURCE.WebApp]
 SUPPORTED_CLIENT_TYPE[RESOURCE.Local] = SUPPORTED_CLIENT_TYPE[RESOURCE.WebApp]
 SUPPORTED_CLIENT_TYPE[RESOURCE.FunctionApp] = SUPPORTED_CLIENT_TYPE[RESOURCE.WebApp]
+
+
+# The dict defines the options for opt-out
+class OPT_OUT_OPTION(Enum):
+    PUBLIC_NETWORK = 'publicnetwork'
+    CONFIGURATION_INFO = 'configinfo'
```

Comment:
how about just using `network` and `appsettings` ?

Reply:
`network` sounds fine to me, though it sounds like private endpoint solution is involved. `firewall` is another option, what do you think?
`appsettings` might be a little exclusive to web app?